omod/src/main/webapp/pages/clinicianfacing/patient.gsp

```diff
@@ -48,9 +48,8 @@
         // create a base map from the fragmentConfig if it exists, otherwise just create an empty map
         def configs = [:];
         if(it.extensionParams.fragmentConfig != null){
-            configs = it.extensionParams.fragmentConfig;
```

Comment:
Ah, so this is just assigning the config to point to the extension paramaters instead of copying.